src/Car.java

```diff
@@ -0,0 +1,6 @@
+public class Car extends Transport {
```

Comment:
```suggestion
public class Car extends MotorTransport {
```
Автомобиль имеет движок. Поэтому наследуем MotorTransport